examples/nextjs-todo/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "easy-peasy-example-nextjs-todo",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "easy-peasy": "^5.1.0",
```

Comment:
Can we make these local references? Might help when doing local dev as will have a whole bunch of app cases we can run and test?

Reply:
That would be ideal - but I'll have to read up on how to do it. I'll play around with it 👍

Reply:
On second thought, when using `file:../../` - the install step takes ages. I'm not sure why - but because "examples" is a subfolder of the package - it might be some recursion madness going on when installing dependencies.

```bash
❯ yarn
yarn install v1.22.19
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Saved lockfile.
Done in 143.50s.
```

It is increasing for each "example" project added 🤔

In contrast, installs with the npm version takes 33 seconds.